### PR TITLE
Protected Routes - component stub up

### PR DESCRIPTION
**Pull request by dcollis92**
### **Protected Routes**

Per Ryan's issue list, here is the stub up of the Protected Routes component, very similar to what we've used in our prior React projects. Currently is commented out to prevent React errors for unused variables.

#### LEFT TO DO:
- (Possibly) need to alter component based on current user params. 
- once the Protected Route component is wrapping the desired Route elements (new, edit), the `user={user}` crosscheck needs to be adjusted. 
     - Current functionality makes additional fetch requests for each Tab, when said user info is already stored in the tab_id. 
     - This adjustment will prevent additional AWS calls
- Need to adjust console.logs to find user id information based off of the existing tab_id in the TablatureProvider.

### CODE REVIEW

File: src/pages/Home/components/ContentRoutes/ContentRoutes.js [hunk 1]
```diff
@@ -3,30 +3,37 @@ import { Route, Routes, Navigate } from "react-router-dom";
 import ProfileContent from "../ProfileContent/ProfileContent";
 import LoginSignupForm from "../LoginSignupForm/LoginSignupForm";
 import Editor from "../Editor/Editor";
+import ProtectedEditRoute from "../Editor/components/ProtectedEditRoute/ProtectedEditRoute";
+import { useContext } from "react";
+import { UserContext } from "containers/CognitoUserProvider/CognitoUserProvider";
```
Comment (RyanSUP) [inline src/pages/Home/components/ContentRoutes/ContentRoutes.js:8]: Doesn't look like `user` is being used anywhere in ContentRoutes, just being passed to ProtectedEditRoute. Why not just import this data in ProtectedEditRoute? 

Was it your intention to set this up to address issue #53 ?

File: src/pages/Home/components/Editor/components/NoteTextarea/NoteTextarea.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ const NoteTestarea = (props) => {
 
   return (
       <TextareaAutosize
-        spellcheck="false"
+        spellCheck="false"
```
Comment (RyanSUP) [inline src/pages/Home/components/Editor/components/NoteTextarea/NoteTextarea.js:29]: Thanks! That error was annoying. (There's a typo in the component name too, but I fixed that on my dev-branch)

**Review by RyanSUP — COMMENTED** [review]
Looks pretty good, thanks for getting this running! We'll need to address #53 before we can really test the functionality, but this puts us in a good spot. Much appreciated. 

Check out the comment I left in ContentRoute diff.

File: src/pages/Home/components/ContentRoutes/ContentRoutes.js (hunk 1 above)
Comment (dcollis92) [reply src/pages/Home/components/ContentRoutes/ContentRoutes.js:8]: I think this was done during our troubleshooting prior to the commit. I remember you suggested it and I had proposed it originally, as a way to identify the user and have that date passed through the component. Feel free to remove it or we can keep it in place until #53 is addressed.

Comment (dcollis92) [reply src/pages/Home/components/ContentRoutes/ContentRoutes.js]: just updated!

**Review by RyanSUP — APPROVED** [review]

**Pull request merged**